rewrite-gradle/src/main/java/org/openrewrite/gradle/AddDependencyVisitor.java

```diff
@@ -16,6 +16,8 @@
 package org.openrewrite.gradle;
 
 import lombok.RequiredArgsConstructor;
+
+import org.codehaus.groovy.syntax.Types;
```

Comment:
I noticed this import was the only one outside of openrewrite and java, is this considered bad form? Where would you recommend putting this dependency?